### PR TITLE
[QA-1696] Fix click create flakiness

### PR DESCRIPTION
**Pull request by xuscrbw**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>
Testing this fix out to see if this helps to resolve the terra-ui tests.

After testing locally, it seems like the test gets stuck when clicking the create button after: creating notebook > edit >Create runtime.

It might pass more consistently if we slightly increase the delay before the page opens.

### CODE REVIEW

**Comment by xuscrbw** [comment]
multi-test

File: integration-tests/tests/run-notebook.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const testRunNotebookFn = _.flow(
   await select(page, 'Language', 'Python 3')
   await click(page, clickable({ text: 'Create Notebook' }))
   await click(page, clickable({ textContains: notebookName }))
+  await delay(2000)
```
Comment (breilly2) [inline integration-tests/tests/run-notebook.js]: Do you need this new delay or is increasing the delay below enough? Adding delays in a test is not ideal. When necessary, it should be justified like the comment below justifies the existing delay. Without that, someone coming across this in the future has no idea if the amount of delay is still appropriate or if it's even needed at all.

File: integration-tests/tests/run-notebook.js [hunk 2]
```diff
@@ -40,7 +41,7 @@ const testRunNotebookFn = _.flow(
   // 500ms delay in the test is not enough to guarantee that the UI elements have finished moving.
   // Therefore, we start with a 1000ms delay, then make sure there are no spinners just in case the
```
Comment (breilly2) [inline integration-tests/tests/run-notebook.js]: Nit: update comment to reflect new learning about the delay length.

Comment (kyuksel) [reply integration-tests/tests/run-notebook.js]: +1

File: integration-tests/tests/run-notebook.js (hunk 1 above)
Comment (kyuksel) [reply integration-tests/tests/run-notebook.js]: 💯

**Review by kyuksel — APPROVED** [review]
LGTM pending @breilly2's comments.

I'd also suggest running the test ~10 times to have a statistically better idea about the stability of the test. In that vein, I just took the liberty to kick off a re-run.

**Comment by kyuksel** [comment]
@xuscrbw I've rerun `run-notebook` several times through the CI and it passed every time  🤩

File: integration-tests/tests/run-notebook.js (hunk 2 above)
Comment (xuscrbw) [reply integration-tests/tests/run-notebook.js]: Addressed

File: integration-tests/tests/run-notebook.js (hunk 1 above)
Comment (xuscrbw) [reply integration-tests/tests/run-notebook.js]: I've been having more success with this new delay locally, but going to try a few runs without it on this PR.

**Comment by xuscrbw** [comment]
@breilly2 Re-requested a review. I was able to get two successful test passes without the first delay and only adding an increase to the second. Also addressed your nit comment

**Review by breilly2 — APPROVED** [review]
👍  Thanks for addressing my feedback!

**Comment by breilly2** [comment]
Hmmm... I wouldn't expect this delay to need to be 3 seconds. This delay was just to cover the gap between the drawer sliding in and the spinner to render (while loading cloud environment details). Makes me think that something else is going on here.

**Comment by xuscrbw** [comment]
> Hmmm... I wouldn't expect this delay to need to be 3 seconds. This delay was just to cover the gap between the drawer sliding in and the spinner to render (while loading cloud environment details). Makes me think that something else is going on here.

I reverted the increase and it looks like it managed to pass the last 6 runs. I am going to merge this, just to unblock things for now. May keep looking for other answers in the meantime

**Comment by kyuksel** [comment]
> I reverted the increase and it looks like it managed to pass the last 6 runs. I am going to merge this, just to unblock things for now. May keep looking for other answers in the meantime

Sounds good. Thanks for looking into it.

**Pull request merged**